afe/main.go

```diff
@@ -1,22 +1,33 @@
 package main
 
 import (
+	"flag"
 	"fmt"
-	"log"
+	"log/slog"
+	"net"
 	"os"
 	"time"
 
 	"github.com/IBM/sarama"
+	"github.com/IRIO-ORG/Trading-System/common"
+	pb "github.com/IRIO-ORG/Trading-System/proto"
+	"google.golang.org/grpc"
 )
 
-func main() {
+var (
+	grpcBindAddress = flag.String("bind_address", "", "Address of the server interface")
+	grpcPort        = flag.Int("grpc_port", 50051, "The gRPC server port")
```

Comment:
It is easier to chang env variables than to modify command line arguments in kubernetes